composer.json

```diff
@@ -30,6 +31,7 @@
         "laravel/ui": "^4.2",
         "league/flysystem-aws-s3-v3": "^3.0",
         "livewire/livewire": "^3.0@beta",
+        "owen-it/laravel-auditing": "^13.5",
```

Comment:
Should this be installed on the module instead of the root?

Reply:
Hmm, that should probably work and probably be how it works. Let me try